src/Compiler/Driver/CompilerConfig.fs

```diff
@@ -1131,13 +1131,7 @@ type TcConfig private (data: TcConfigBuilder, validate: bool) =
                         yield clrFacades
 
                 | None ->
-                    // "there is no really good notion of runtime directory on .NETCore"
-#if NETSTANDARD
                     let runtimeRoot = Path.GetDirectoryName(typeof<Object>.Assembly.Location)
```

Comment:
System.Runtime.InteropServices.RuntimeEnvironment.GetRuntimeDirectory() exists on netstandard 2.0.  It is probably a better solution than location of assembly containing int.